docs/ConnectPOB/manage-nodes.md

```diff
@@ -10,15 +10,72 @@ NodeAdmin show all nodes in a table and where they are placed on the map. This i
+|     I     | Shows number of personell that is registered as embarked with wearables in the ScanReach system.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | ![POB count](./node-admin-images/toolbar-images/pob-count.png)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
Fixed some typos and shortened some text.

Would also be nicer if we extracted the icons from Figma without the backgrounds?
```suggestion
| List item | Explanation                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            | Variations                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
| :-------: | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
|     A     | Search for nodes by mac address. The number of nodes visible in the map (GA - General Arrangement) and the [node list](#node-list) will decrease as a user types the in the search bar.                                                                                                                                                                                                                                                                                                                                                                | NIL                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
|     B     | Filter toggle for placed nodes in the map.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | <ul><li class="list-no-bullet-points">![Filtering placed nodes](./node-admin-images/toolbar-images/placed-nodes-show.png)Default on - Shows nodes that is placed in the map</li><li class="list-no-bullet-points">![Filtering placed nodes](./node-admin-images/toolbar-images/placed-nodes-hide.png)Placed nodes is now hidden in map.</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
|     C     | Filter toggle for not placed nodes in the map.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | <ul><li class="list-no-bullet-points">![Filtering not placed nodes](./node-admin-images/toolbar-images/not-placed-nodes-show.png)Default on - Shows nodes that is NOT placed in the map</li><li class="list-no-bullet-points">![Filtering not placed nodes](./node-admin-images/toolbar-images/not-placed-nodes-hide.png)NOT placed nodes is now hidden in map.</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
|     D     | Filter toggle for nodes configured as musterstations in the map. More on musterstation configuration [here](#node-manager).                                                                                                                                                                                                                                                                                                                                                                                                                            | <ul><li class="list-no-bullet-points">![Musterstation nodes](./node-admin-images/toolbar-images/musterstations-show.png)Default on - Shows nodes that is configured as musterstations in th map with the musterstation icon ![Musterstation icon](../icons/muster.svg)</li><li class="list-no-bullet-points">![Musterstation nodes](./node-admin-images/toolbar-images/musterstations-hide.png)Nodes configured as musterstations is now hidden in map.</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
|     E     | Filter toggle for node links in the map. More on node links [here](#showing-node-parent).                                                                                                                                                                                                                                                                                                                                                                                                                                                              | <ul><li class="list-no-bullet-points">![Node links](./node-admin-images/toolbar-images/node-links-show.png)Default on - Shows the lines between nodes depending on what is selected it shows Parent, children or neighbours</li><li class="list-no-bullet-points">![Node links](./node-admin-images/toolbar-images/node-links-hide.png)Node links is now hidden in map.</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
|     F     | System status. This is dependent on active system alarms and how critical these are: <ul><li>`OK` - everything is functioning well.</li><li>`Failure` - Some aspect of the system has stopped working or is not responding, this should be investigated. The system will most likely still function well.</li><li>`Critical Failure` - There is a critical error in the system that will stop core functionality of the system to work. This error should be handled immediately or contact ScanReach support for assistance in handling it.</li></ul> | <ul><li class="list-no-bullet-points">![System status ok](./node-admin-images/toolbar-images/system-status-ok.png)</li><li class="list-no-bullet-points list-with-extra-padding">![System status ok tablet](./node-admin-images/toolbar-images/system-status-ok-tablet.png)System status: OK</li><li class="list-no-bullet-points">![System status failure](./node-admin-images/toolbar-images/system-status-failure.png)</li><li class="list-no-bullet-points list-with-extra-padding">![System status failure tablet](./node-admin-images/toolbar-images/system-status-failure-tablet.png)System status: Failure</li><li class="list-no-bullet-points">![System status critical failure](./node-admin-images/toolbar-images/system-status-critical-failure.png)</li><li class="list-no-bullet-points">![System status critical failure tablet](./node-admin-images/toolbar-images/system-status-critical-failure-tablet.png)System status: Critical Failure</li></ul> |
|     G     | Mute button to mute audible alarms from the digital output node. More on the digital output node [here](#node-manager).                                                                                                                                                                                                                                                                                                                                                                                                                                | <ul><li class="list-no-bullet-points">![Mute button active](./node-admin-images/toolbar-images/mute-active.png)Possible to press button to send mute command</li><li class="list-no-bullet-points">![mute button disabled](./node-admin-images/toolbar-images/mute-disable.png)Mute button is disabled when system is sending mute command</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
|     H     | Mustering toggle to initiate or deactivate Mustering in the ScanReach system. More on Mustering [here](mustering.md).                                                                                                                                                                                                                                                                                                                                                                                                                                  | <ul><li class="list-no-bullet-points">![Mustering off](./node-admin-images/toolbar-images/mustering-off.png)</li><li class="list-no-bullet-points list-with-extra-padding">![Mustering off tablet](./node-admin-images/toolbar-images/mustering-off-tablet.png)Mustering is off</li><li class="list-no-bullet-points">![Mustering on](./node-admin-images/toolbar-images/mustering-on.png)</li><li class="list-no-bullet-points">![Mustering on](./node-admin-images/toolbar-images/mustering-on-tablet.png)Mustering is initiated</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                            |
|     I     | Embarked personnel in the ScanReach system.    
| :-------: | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
|     A     | A user can search for a node by the given name of a node or the mac address of a node. The number of nodes visible in the map (GA - General Arrangement) and the [node list](#node-list) will decrease as a user types the in the search bar. Until only one node matches the searh parameter.                                                                                                                                                                                                                                                                                                                         | NIL                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
|     B     | Filtering for showing or hiding placed nodes in the map.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | <ul><li class="list-no-bullet-points">![Filtering placed nodes](./node-admin-images/toolbar-images/placed-nodes-show.png)Default on - Shows nodes that is placed in the map</li><li class="list-no-bullet-points">![Filtering placed nodes](./node-admin-images/toolbar-images/placed-nodes-hide.png)Placed nodes is now hidden in map.</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
|     C     | Filtering for showing or hiding not placed nodes in the map.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | <ul><li class="list-no-bullet-points">![Filtering not placed nodes](./node-admin-images/toolbar-images/not-placed-nodes-show.png)Default on - Shows nodes that is NOT placed in the map</li><li class="list-no-bullet-points">![Filtering not placed nodes](./node-admin-images/toolbar-images/not-placed-nodes-hide.png)NOT placed nodes is now hidden in map.</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
|     D     | Filtering for showing or hiding nodes configured to be musterstations in the map. More on musterstation configuration [here](#node-manager).                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | <ul><li class="list-no-bullet-points">![Musterstaion nodes](./node-admin-images/toolbar-images/musterstations-show.png)Default on - Shows nodes that is configured as musterstations in th map with the musterstation icon ![Musterstation icon](../icons/muster.svg)</li><li class="list-no-bullet-points">![Musterstation nodes](./node-admin-images/toolbar-images/musterstations-hide.png)Nodes configured as musterstations is now hidden in map.</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
|     E     | Filtering for showing or hiding node links in the map. These are either dotted when not selected or full drawn when selecting a node More on node links [here](#showing-node-parent).                                                                                                                                                                                                                                                                                                                                                                                                                                  | <ul><li class="list-no-bullet-points">![Node links](./node-admin-images/toolbar-images/node-links-show.png)Default on - Shows the lines between nodes dependign on what is selected it shows Parent, children or neighbours</li><li class="list-no-bullet-points">![Node links](./node-admin-images/toolbar-images/node-links-hide.png)Node links is now hidden in map.</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
|     F     | Shows the system status. This is dependent on if there are system alarms and how system critical these are. <ul><li> System status: OK - everything is functioning well.</li><li> System status: Failure - Some aspect of the system has stopped working or is not responding, this should be investigated. The system will most likely still function well.</li><li> System status: Critical Failure - There is a critical error in the system that will stop core functionality of the system to work. This error should be handled immediatly or contact ScanReach support for assistance in handling it.</li></ul> | <ul><li class="list-no-bullet-points">![System status ok](./node-admin-images/toolbar-images/system-status-ok.png)</li><li class="list-no-bullet-points list-with-extra-padding">![System status ok tablet](./node-admin-images/toolbar-images/system-status-ok-tablet.png)System status: OK</li><li class="list-no-bullet-points">![System status failure](./node-admin-images/toolbar-images/system-status-failure.png)</li><li class="list-no-bullet-points list-with-extra-padding">![System status failure tablet](./node-admin-images/toolbar-images/system-status-failure-tablet.png)System status: Failure</li><li class="list-no-bullet-points">![System status critical failure](./node-admin-images/toolbar-images/system-status-critical-failure.png)</li><li class="list-no-bullet-points">![System status critical failure tablet](./node-admin-images/toolbar-images/system-status-critical-failure-tablet.png)System status: Critical Failure</li></ul> |
|     G     | Mute button to mute audible alarms from the digital output node. More on the digital output node [here](#node-manager).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | <ul><li class="list-no-bullet-points">![Mute button active](./node-admin-images/toolbar-images/mute-active.png)Possible to press button to send mute command</li><li class="list-no-bullet-points">![mute button disabled](./node-admin-images/toolbar-images/mute-disable.png)Mute button is disabled when system is sending mute command</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
|     H     | A toggle to initiate or deactivate Mustering in the ScanReach system. More on Mustering [here](mustering.md).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | <ul><li class="list-no-bullet-points">![Mustering off](./node-admin-images/toolbar-images/mustering-off.png)</li><li class="list-no-bullet-points list-with-extra-padding">![Mustering off tablet](./node-admin-images/toolbar-images/mustering-off-tablet.png)Mustering is off</li><li class="list-no-bullet-points">![Mustering on](./node-admin-images/toolbar-images/mustering-on.png)</li><li class="list-no-bullet-points">![Mustering on](./node-admin-images/toolbar-images/mustering-on-tablet.png)Mustering is initiated</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                            |
|     I     | Shows number of personell that is registered as embarked with wearables in the ScanReach system.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | ![POB count](./node-admin-images/toolbar-images/pob-count.png)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```